tasks/config/simplemocha.js

```diff
@@ -22,6 +22,9 @@ export default {
       '!**/__tests__/fixtures/**/*',
       '!src/**/public/**',
       '!**/_*.js',
+
+      // These tests have been moved to Jest
+      '!src/server/**/*.js',
```

Comment:
Unless we're going to lint to make sure there aren't mocha test files in the server code I don't think we should negate like this. I just merged a PR that adds some new tests that aren't migrated here, and I expect that it would be easy for people to test individual test files and not realize they are not being run by simplemocha.